context.go

```diff
@@ -2,14 +2,13 @@ package onet
 
 import (
 	"errors"
-	"fmt"
-	"io/ioutil"
 	"os"
 	"os/user"
 	"path"
 	"runtime"
 	"sync"
 
+	bolt "github.com/coreos/bbolt"
```

Comment:
Is this the standard import for bbolt? Else using `bbolt.` is not too bad, neither. I prefer not overwriting the package name in general.

Reply:
This is what is used in all the bbolt example (https://github.com/coreos/bbolt). I'm guessing it's because bbolt is a fork of bolt, but to maintain compatibility, they rename the import to bolt.